app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class MenusController < ApplicationController
+  before_action :set_menu, only: %i[show edit update destroy]
+
+  def index
+    @menus = Menu.all
+    @cart_item = current_cart.cart_items.new
+  end
+
+  def show; end
+
+  def new
+    @menu = Menu.new
+  end
+
+  def create
+    @menu = Menu.new(menu_params)
+    if @menu.save
+      redirect_to admin_infobase_index_path
+    else
+      admin_infobase_index_path
```

Comment:
I wrote anything else, because I'm not sure this like is working at all :confused: 
even if it is, that's a weird way. Please consider using `render` with specifing needed action